test/stdlib/DispatchTypes.swift

```diff
@@ -1,6 +1,7 @@
 // RUN: %target-swift-frontend -typecheck %s
 
 // REQUIRES: libdispatch
+// XFAIL: OS=linux-gnu
```

Comment:
```
/home/yln/work/swift-source/swift/test/stdlib/DispatchTypes.swift:27:45: error: use of unresolved identifier 'mach_port_t'
_ = DispatchSource.makeMachSendSource(port: mach_port_t(0), eventMask: [])
                                            ^~~~~~~~~~~
/home/yln/work/swift-source/swift/test/stdlib/DispatchTypes.swift:28:48: error: use of unresolved identifier 'mach_port_t'
_ = DispatchSource.makeMachReceiveSource(port: mach_port_t(0))
                                               ^~~~~~~~~~~
/home/yln/work/swift-source/swift/test/stdlib/DispatchTypes.swift:29:20: error: type 'DispatchSource' has no member 'makeMemoryPressureSource'
_ = DispatchSource.makeMemoryPressureSource(eventMask: [])
    ~~~~~~~~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~~
/home/yln/work/swift-source/swift/test/stdlib/DispatchTypes.swift:30:20: error: type 'DispatchSource' has no member 'makeProcessSource'; did you mean 'makeWriteSource'?
_ = DispatchSource.makeProcessSource(identifier: 0, eventMask: [])
    ~~~~~~~~~~~~~~ ^~~~~~~~~~~~~~~~~
                   makeWriteSource
Dispatch.DispatchSource:32:23: note: 'makeWriteSource' declared here
    public class func makeWriteSource(fileDescriptor: Int32, queue: DispatchQueue? = nil) -> DispatchSourceWrite
                      ^
/home/yln/work/swift-source/swift/test/stdlib/DispatchTypes.swift:32:45: error: use of unresolved identifier 'SIGINT'
_ = DispatchSource.makeSignalSource(signal: SIGINT)
                                            ^~~~~~
/home/yln/work/swift-source/swift/test/stdlib/DispatchTypes.swift:34:20: error: type 'DispatchSource' has no member 'makeFileSystemObjectSource'
_ = DispatchSource.makeFileSystemObjectSource(fileDescriptor: 0, eventMask: [])
    ~~~~~~~~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Sounds like `-DDEPLOYMENT_TARGET_LINUX` may be missing?